Gemfile

```
@@ -4,5 +4,6 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem "rake", "~> 13.1.0"
```

I'd pin it to a .y version to allow updates to 13.2 for example.
```suggestion
gem "rake", "~> 13.1"
```